pkg/resource/aws/aws_route53_record_acceptance_test.go

```diff
@@ -0,0 +1,29 @@
+package aws_test
+
+import (
+	"testing"
+
+	"github.com/cloudskiff/driftctl/test/acceptance"
+)
+
+func TestAcc_AwsRoute53Record_WithFQDNAsId(t *testing.T) {
```

Comment:
You can remove this duplicated ACC test, it is already done in `aws_route53_record_test.go`

This PR should not modify Acceptance tests

Reply:
It was not a duplicate test, i just moved acc test in another file to create new tests for route53_record_ext but this is canceled since we explain me we keep test in their standard file.